irj_parser.opam

```diff
@@ -1,22 +1,18 @@
 # This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
+version: "1.1.0"
 synopsis: "Parser for the IRJ tests"
 description:
   "This parser is aimed for the tests used by la DGFiP to test the calculation of the French income tax"
-maintainer: ["Mathieu Durero" " Noé Ensarguet"]
```

Comment:
Ha ha why did you remove yourself and Mathieu and put me as the maintainer ;)

Reply:
Hahaha
I had a separate dune-project for irj_parser in the first commit, until I realised (thanks to @Keryan-dev) that you could declare both packages in the same dune-project file (and so with the same maintainers etc.)